### DISCUSSION TITLE
Trojan:Win32/Vigorf.A

### DISCUSSION THREAD

**Discussion by alcio313** [xournalpp/xournalpp#5383] [Q&A]
Windows Defender delete the zip installer because it says found this Trojan, does anyone have this issue?
https://github.com/xournalpp/xournalpp/releases/download/v1.2.2/xournalpp-1.2.2-windows.zip

**Comment by NiklasPaschukos** [comment]
I have just experienced the same issue. For fear of further malware being present I am now hesitant to try and download previous builds to compare.

**Reply by rolandlo** [reply]
This is closely related to https://github.com/xournalpp/xournalpp/issues/5378. Let's continue discussing the problem there.
